tasks/test_syntax.js

```diff
@@ -188,6 +190,47 @@ function assertCircularDeps() {
     });
 }
 
+// Ensure no ES6 has snuck through into the build:
+function assertES5() {
+    var CLIEngine = eslint.CLIEngine;
+
+    var cli = new CLIEngine({
+        useEslintrc: false,
+        ignore: false,
+        parserOptions: {
+            ecmaVersion: 5
+        }
+    });
+
+    // Filter out min and plotly-geo-assets.js since one is unnecessary
+    // and the other is super slow:
+    var files = fs.readdirSync(path.join(__dirname, '../dist'));
+    var validFiles = [];
+    for(var i = 0; i < files.length; i++) {
+        var f = files[i];
+        var isMin = !/[^(min)]\.js$/.test(f);
```

Comment:
Why not just use this [list](https://github.com/plotly/plotly.js/blob/master/tasks/util/constants.js#L19), something like:

```js
var validFiles = constants.partialBundlePaths(function(p) { return p.dist; });
validFiles  = validFiles.concat(constants.pathToPlotlyDist)l

var report = cli.executeOnFiles(validFiles);
// ...
```

looks easier to read to me. Thoughts?